### PR TITLE
User-facing errors docs

### PR DESCRIPTION
**Pull request by lemonez**
[DOCS-189](https://algorithmia.atlassian.net/browse/DOCS-189) - DC docs for algorithm errors for non-admin accounts

Also rewrote the algo profile and account profile pages where the errors info goes, because they were super outdated.

I didn't update any old screenshots.

## Checklist
- [x] My PR title includes a relevant Jira ticket name
- [ ] If no associated Jira ticket, my PR title is informative (individual commit messages are squashed in this repo)
- [x] Unnecessary text (notes, TODOs, etc.) has been removed
- [x] For YAML "front matter":
  - Properties are alphabetized
  - `permalink` and `redirect_from/to` properties have `/` at beginning and end (e.g., `/path/`)
- [x] Screenshots, if present, follow the [Screenshot guidelines](https://algorithmia.atlassian.net/wiki/spaces/CUSTOMERS/pages/1634861478/CFD+Style+Guide#Screenshots) which means:
  - Full browser width (browser at 100% zoom and 10-12" width)
  - Max 1000px
- [x] Grammar, style, tone, punctuation, etc. follow the Algorithmia [manual of style](https://docs.google.com/document/d/1PPVfgMkX7-EVGLPMhN1E485CAXu9QfhSLKM0lZhnZdU/edit?usp=sharing)
- [x] Headers are “Sentence case with Proper Nouns capitalized”
- [x] Placeholder strings are `UPPER_SNAKE_CASE` and are named as in the [Glossary](https://docs.google.com/document/d/1bYs0j_a4v8LbkbS8r0x2SZ_J6mZ4sIt4w_60dq8iGh4/edit#heading=h.qov2yks2a685) where appropriate

### CODE REVIEW

**Comment by lemonez** [comment]
@StephanieKim or @warmbowski I told Chris this would be in by EOW so I'm hoping for a review today/tomorrow, thanks!

File: _pages/platform/account-profile.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+
+The **number of calls** column lists the number of times that you've called each algorithm, but you'll see that sometimes the **billed calls** column displays a lower value. This is because we don't bill API calls that result in an error.
+{: .notice-info}
+
+The **billed time** column lists the time billed, in seconds, which is the time it took for the algorithm to execute on the platform. We use this compute time and the royalty on the algorithm to calculate the value in the **cost in credits** column.
+![Algorithm usage]({{site.cdnurl}}{{site.baseurl}}/images/post_images/your_profile/usage.png)
+
+For more information about pricing, see [Pricing]({{site.baseurl}}/pricing).
+
+#### Earnings
```
Comment (warmbowski) [inline _pages/platform/account-profile.md:81]: I don't think this exists on the user page.

Comment (lemonez) [reply _pages/platform/account-profile.md:81]: which part doesn't exist?

image

Comment (warmbowski) [reply _pages/platform/account-profile.md:81]: ah tabs under tabs. I get it now.

**Review by warmbowski — APPROVED** [review]
this lgtm

**Comment by lemonez** [comment]
> this lgtm

me too now that i have it up and running 👍🏼

**Pull request merged**